src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -114,6 +115,35 @@ def try_path_munging(
 
 @region_silo_endpoint
 class ProjectStacktraceLinkEndpoint(ProjectEndpoint):  # type: ignore
+    """
+    Returns the index in which the new code mapping should be inserted in the code mapping list.
+    User generated code mappings are evaluated before Sentry generated code mappings.
+    Code mappings with more defined stack trace roots are evaluated before less defined stack trace
+    roots.
+
+    `matched_code_matchings`: The list of code mappings
+    `new_code_mapping`: The new code mapping that is being inserted
+    """
+
+    def get_matched_code_mapping_index(self, matched_code_mappings, new_code_mapping):
```

Comment:
Function to return the index in which the new code mapping should be inserted in the list. This will determine the order that the code mappings are evaluated